### PR TITLE
Update mainnet discount validator contract addresses

### PR DESCRIPTION
**Pull request by dneilroth**
**What changed? Why?**
Update mainnet discount validator contract addresses
**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/web/src/addresses/usernames.ts [hunk 1]
```diff
@@ -20,17 +20,17 @@ export const USERNAME_EA_REGISTRAR_CONTROLLER_ADDRESSES: AddressMap = {
 
 export const USERNAME_CB_ID_DISCOUNT_VALIDATORS: AddressMap = {
   [baseSepolia.id]: '0xA316d8bcEC25587a1Fad599ae6B56169cf377816',
-  [base.id]: '0x',
+  [base.id]: '0x0A484e560946818787135EAD632771589523dE82',
```
Comment (dneilroth) [inline apps/web/src/addresses/usernames.ts:23]: https://basescan.org/address/0x0A484e560946818787135EAD632771589523dE82

File: apps/web/src/addresses/usernames.ts [hunk 2]
```diff
@@ -20,17 +20,17 @@ export const USERNAME_EA_REGISTRAR_CONTROLLER_ADDRESSES: AddressMap = {
...
 };
 
 export const USERNAME_CB1_DISCOUNT_VALIDATORS: AddressMap = {
   [baseSepolia.id]: '0xcf3078c5f2C5d8B7E7CF1a23dD7Bb50244423273',
-  [base.id]: '0x',
+  [base.id]: '0x9de4Ab12320684cec803Edb72aA3a920250d392C',
```
Comment (dneilroth) [inline apps/web/src/addresses/usernames.ts:28]: https://basescan.org/address/0x9de4Ab12320684cec803Edb72aA3a920250d392C

File: apps/web/src/addresses/usernames.ts [hunk 3]
```diff
@@ -20,17 +20,17 @@ export const USERNAME_EA_REGISTRAR_CONTROLLER_ADDRESSES: AddressMap = {
...
 };
 
 export const USERNAME_CB_DISCOUNT_VALIDATORS: AddressMap = {
   [baseSepolia.id]: '0x949dB534C2D5F777c7755fF22111B01934C87556',
-  [base.id]: '0x',
+  [base.id]: '0x012076854d030128dc72B34621287Bb585210315',
```
Comment (dneilroth) [inline apps/web/src/addresses/usernames.ts:33]: https://basescan.org/address/0x012076854d030128dc72B34621287Bb585210315

File: apps/web/src/addresses/usernames.ts [hunk 4]
```diff
@@ -40,17 +40,17 @@ export const USERNAME_EA_DISCOUNT_VALIDATORS: AddressMap = {
 
 export const USERNAME_BNS_DISCOUNT_VALIDATORS: AddressMap = {
   [baseSepolia.id]: '0x1DE649d8b004A44491a7D3ebbb23F4B0DA89DE78',
-  [base.id]: '0x',
+  [base.id]: '0x20b433c640DFb8c2e3C6aBB0533314b2d7B9f2FF',
```
Comment (dneilroth) [inline apps/web/src/addresses/usernames.ts:43]: https://basescan.org/address/0x20b433c640DFb8c2e3C6aBB0533314b2d7B9f2FF

File: apps/web/src/addresses/usernames.ts [hunk 5]
```diff
@@ -40,17 +40,17 @@ export const USERNAME_EA_DISCOUNT_VALIDATORS: AddressMap = {
...
 };
 
 export const BASE_DOT_ETH_ERC721_DISCOUNT_VALIDATOR: AddressMap = {
   [baseSepolia.id]: '0xa0C7a114E25618538BE7fA7c6552C3122056F775',
-  [base.id]: '0x',
+  [base.id]: '0x55564490a44FDC2aEEa54B60eB1c79F124FD88b9',
```
Comment (dneilroth) [inline apps/web/src/addresses/usernames.ts:48]: https://basescan.org/address/0x55564490a44FDC2aEEa54B60eB1c79F124FD88b9

File: apps/web/src/addresses/usernames.ts [hunk 6]
```diff
@@ -40,17 +40,17 @@ export const USERNAME_EA_DISCOUNT_VALIDATORS: AddressMap = {
...
 };
 
 export const BUILDATHON_ERC721_DISCOUNT_VALIDATOR: AddressMap = {
   [baseSepolia.id]: '0x7b5B2dB59c414e15Bf70b59C02E6fb00Ca919dbC',
-  [base.id]: '0x',
+  [base.id]: '0xB635802085b405A9C8BA7225ae866f60b63d8503',
```
Comment (dneilroth) [inline apps/web/src/addresses/usernames.ts:53]: https://basescan.org/address/0xB635802085b405A9C8BA7225ae866f60b63d8503

**Review by stevieraykatz — APPROVED** [review]

**Review by zencephalon — APPROVED** [review]

**Pull request merged**